### PR TITLE
updated footer, changed spacing

### PR DESCRIPTION
**Pull request by saikrishnatbijil**
made footer more good by adjusting some spacing 
**before:**
Screen Shot 2023-04-14 at 8 19 29 PM
**after:**
Screen Shot 2023-04-14 at 8 19 06 PM
now it looks more good and gives more space to breath.

### CODE REVIEW

**Review by argshook — CHANGES_REQUESTED** [review]
Hey, thanks for contribution!

We can add this change but the suggested code is too bulky for a little tweak.

All that's needed for this layout is to use these rules on the `div` in line `123`:

```
display: grid;
grid-template-columns: auto min-content min-content;
```

please add it as a new class and we can merge!

**Comment by saikrishnatbijil** [comment]
@argshook requested changes are added

**Comment by argshook** [comment]
@saikrishnatbijil 

> @argshook requested changes are added

thanks! sorry i should have mentioned, now your previously done changes should be reverted. For example, there's no longer class name like `styles.margin` that you added previously

**Comment by saikrishnatbijil** [comment]
@argshook i have added margin back

**Comment by argshook** [comment]
@saikrishnatbijil 

pardon me for not being clear enough!

the code should be like so:

at the bottom of `src/theme/Footer/index.tsx`:
```
      <div className={clsx(styles.border, styles.bottom)}>
        {customFields.copyright}
        <a className={styles.link} href="/privacy-notice/">
          Privacy
        </a>
        <a className={styles.link} href="/terms/">
          Terms
        </a>
      </div>
```

and `src/theme/Footer/styles.module.css`:
```
.bottom {
  display: grid;
  grid-template-columns: auto min-content min-content;
  align-items: center;
  padding: 1rem 1rem 2rem;
  gap: 2rem;
  color: var(--palette-white);
}
```

and that's it! The desired effect should be done.

Please notice how the only `className` used is `styles.bottom`, there's no `styles.spacing` nor `styles.margin`.

This change also reduces the number of nested `div`s which is always a nice thing to do.

**Comment by saikrishnatbijil** [comment]
@argshook i have changed the code that it is more efficient. Thank you for the suggestion

File: src/theme/Footer/styles.module.css [hunk 1]
```diff
@@ -130,7 +130,8 @@
 }
 
 .bottom {
-  display: flex;
+  display: grid;
+  grid-template-columns: auto min-content min-content;
```
Comment (argshook) [inline src/theme/Footer/styles.module.css:134]:
```suggestion
  grid-template-columns: auto min-content min-content;
  max-width: var(--ifm-container-width);
  margin: 0 auto;
```

File: src/theme/Footer/index.tsx [hunk 2]
```diff
@@ -120,14 +120,12 @@ const Footer = () => {
         </div>
       </div>
 
-      <div className={styles.border}>
-        <div className={clsx(styles.bottom, styles.center)}>
+      <div className={clsx(styles.border)}>
+        <div className={clsx(styles.border, styles.bottom)}>
```
Comment (argshook) [inline src/theme/Footer/index.tsx:124]:
```suggestion
      <div className={styles.border}>
        <div classNames={styles.bottom}>
```

**Review by argshook — CHANGES_REQUESTED** [review]
Hey, pardon for being so nit-picky, however, we constantly want to improve code quality, including doing design changes.

I'm putting my code changes as code suggestions, so it should be easy to get them in, there should be no need for you to open a new PR or something.

If you're not familiar with code suggestions, here's a [1 minute guide](https://egghead.io/lessons/github-add-suggestions-in-a-github-pr-review)

**Comment by saikrishnatbijil** [comment]
So sorry @argshook i am just starting out in open source and learning how git and github works. Thank you for your great support. Does everything look good now

**Comment by argshook** [comment]
> So sorry @argshook i am just starting out in open source and learning how git and github works. Thank you for your great support. Does everything look good now

no need to apologise, i think you're doing great!

It looks good, thanks for being patient with my review. I'll merge once build is complete

**Review by argshook — APPROVED** [review]

**Pull request merged**